packages/fx-core/resource/package.nls.json

```diff
@@ -110,8 +110,8 @@
   "core.migrationV3.confirmOnly.Message": "Please confirm the upgrade",
   "core.migrationV3.Message": "Please upgrade your project to stay compatible with the latest version, your current project contains configurations from an older Teams Toolkit.\nThe auto-upgrade process will generate backups in case an error occurs.",
   "core.migrationV3.VS.Message": "The Teams Toolkit preview feature - Teams App Configuration Improvements is turned on. This project is required to be upgraded to work with the preview features. Select \"Learn more\" to read about the required changes or select \"Upgrade\" to perform the changes automatically. If you don't want to upgrade, you can turn off the preview feature.",
-  "core.migrationV3.aadManifestNotExist": "templates/appPackage/aad.manifest.template.json does not exist. You may be trying to upgrade a project created by Teams Toolkit <= v3.8.0. Please install Teams Toolkit v4.x and run upgrade first.",
-  "core.migrationV3.manifestNotExist": "templates/appPackage/manifest.template.json does not exist. You may be trying to upgrade a project created by Teams Toolkit <= v3.8.0. Please install Teams Toolkit v4.x and run upgrade first.",
+  "core.migrationV3.aadManifestNotExist": "templates/appPackage/aad.manifest.template.json does not exist. You may be trying to upgrade a project created by VS Code Teams Toolkit 3.X / TeamsFx CLI 0.X / Visual Studio 2022 17.3. Please install VS Code Teams Toolkit 4.X / TeamsFx CLI 1.X / Visual Studio 2022 17.4 and run upgrade first.",
```

Comment:
VS Code Teams Toolkit 3.X -> Teams Toolkit for Visual Studio Code v3.x
TeamsFx CLI 0.X -> Teams Toolkit CLI v0.x
Visual Studio 2022 17.3 -> Teams Toolkit for Visual Studio v17.3

and apply to the next sentence as well.

Reply:
Updated, thx！